cid/helpers/quicksight/__init__.py

```diff
@@ -211,7 +211,7 @@ def create_data_source(self) -> bool:
         params = {
             "AwsAccountId": self.account_id,
             "DataSourceId": str(uuid.uuid4()),
-            "Name": "Athena",
+            "Name": "Athena-CID",
```

Comment:
Why do you think we should rename datasource?

Reply:
So that we could distinguish once created by us from all other athena datasource in qs

Reply:
This is not needed, please revert

Reply:
not agree but done.